docs/publish/command.py

```diff
@@ -166,5 +164,18 @@ def command(shell: DTShell, args, **kwargs):
                 line = line.decode("utf-8")
                 print(line, end="")
 
-            dtslogger.info(f"Project '{project.name}' published.")
-
+            published_title: str = project.name.replace("book-", "", 1)  #TODO: Where does jupyter-book cut this?
```

Comment:
_Where does jupyter-book cut this?_
here: https://github.com/duckietown/docs.duckietown.com/blob/2876ce261a30198f8eedb9b469cb351f824dadcc/http/books-urlrewrite.location#L4

Reply:
it is never stripped really, it is just rewritten by nginx